experiment/config-forker/README.md

```diff
@@ -29,6 +29,10 @@ For all jobs:
   `master`, the value will be changed to `release-1.15`.
 - If the `fork-per-release-replacements` annotation is specified, those replacements will be performed in the `args`
   of all containers for that job.
+- If the `testgrid-dashboards` annotation is specified, references to `master-blocking` and `master-informing` are
+  changed to `1.15-blocking` and `1.15-informing`.
+- If the `testgrid-tab-name` annotation is specified, references to `master` are changed to `1.15`.
+- If the `description` annotation is specified, it is removed (for now).
```

Comment:
yeah I think I'd like this to get copied along / mangled, but can be in a followup, this more accurately reflects the state of toay

Reply:
So would I, but actually getting it right is tricky.